timetable/templates/timetable_base.html

```diff
@@ -43,9 +43,19 @@ <h3 class="timetable-caption">{% block caption %}{% endblock %}</h3>
 			<tr>
 				<td id="period-{{ period }}">{{ period }}</td>
 				<td>
-					{% autoescape off %}
-					{{ row.0|hyphen_break }}
-					{% endautoescape %}
+					<div style="color: #888888 ;">
+						<!--  Alternative date -->
+						{% if row.2 %} 
+							{% autoescape off %}
+							{{ row.2|hyphen_break }}
+							{% endautoescape %}
+						{% endif %}
+					</div>
+					<div>
+						{% autoescape off %}
+						{{ row.0|hyphen_break }}
+						{% endautoescape %}
+					</div>
```

Comment:
This would work well if you passed the right data retrieved from the db, but I don't think generating empty divs is a good idea. Try not generating them if not necessary.

Reply:
Okay, I will take a look at it.

Reply:
@Igifigi I have fixed it so it creates divs only if there are alternative schedules left